client/extensions/woocommerce/app/order/order-details/table.js

```diff
@@ -261,6 +279,85 @@ class OrderDetailsTable extends Component {
 		);
 	};
 
+	renderCoupons = () => {
+		const { isEditing, order, site, translate } = this.props;
+		const showTax = this.shouldShowTax();
+		const coupons = order.coupon_lines || [];
+		const hasCoupons = parseFloat( order.discount_total ) > 0 || order.coupon_lines.length > 0;
+		if ( ! isEditing && ! hasCoupons ) {
+			return null;
+		}
+
+		if ( isEditing ) {
+			const couponMarkup = coupons.map( item => {
```

Comment:
So... technically no, since this section returns at L306, these two `couponMarkup`s never overlap, but it does make for a double-take while reading the code, so I'll make the change.